samples/languages/python/run.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+export PATH="../../../build:$PATH"
```

Comment:
Should we mention installing sgx-lkl (sudo -E make install) in README and using:
export PATH=/opt//sgx-lkl/bin:$PATH

Reply:
I think this should go into the installation instructions of the root README.